package.json

```diff
@@ -10,9 +10,9 @@
     "type:check": "turbo run type:check",
     "type:emit": "turbo run type:emit",
     "build-storybook": "turbo run build-storybook",
-    "test": "turbo run test",
+    "test": "turbo run test -- --passWithNoTests",
```

Comment:
why do we need this flag?

Reply:
Because without this flag the test command will fail by default on CI due packages which have no tests.

Reply:
and we don't need it in the past, it shouldn't need it now